public/docs/yaml-metadata.md

```diff
@@ -102,35 +102,46 @@ You can only use whether `true` or `false`.
 > default: not set (which will be true)
 
 **Example**
-```xml
+```yml
 breaks: false
 ```
 
 GA
 ---
-This option allow you to enable Google Analytics with your ID.
+This option allows you to enable Google Analytics with your ID.
 
 > default: not set (which won't enable)
 
 **Example**
-```xml
+```yml
 GA: UA-12345667-8
 ```
 
 disqus
 ---
-This option allow you to enable Disqus with your shortname.
+This option allows you to enable Disqus with your shortname.
 
 > default: not set (which won't enable)
 
 **Example**
-```xml
+```yml
 disqus: codimd
 ```
 
+type
+---
+This option allows you to switch the view to the slide mode.
```

Comment:
"view to slide mode" is not very clear. This sounds like activating presentation mode. My own attempts at making this more clear failed, though.